app/src/main/res/values/strings.xml

```diff
@@ -80,9 +80,9 @@
 
   <string name="about_label">About</string>
   <string name="about_powered_by">is powered by</string>
-  <string name="squanchy" translatable="false">Squanchy</string>
+  <string name="squanchy" translatable="false">&#8201;Squanchy&#8201;</string>
   <string name="squanchy_website" translatable="false">https://squanchy.net</string>
-  <string name="about_opensource_blurb">This app is proudly open source and any contribution is welcome.</string>
+  <string name="about_opensource_blurb">This&#160;app&#32;is&#160;proudly&#32;open&#160;source&#32;and&#32;any&#32;contribution&#32;is&#160;welcome.</string>
```

Comment:
can you change `&#160` with `&nbsp`, `&#8201` with `&thinsp` and the `&#32` with normal spaces (since they represent normal spaces)?

It's kind of confusing right now and names are better than numbers :)